engine/lib/phx/script/ffi_ext/TaskQueue.lua

```diff
@@ -0,0 +1,21 @@
+local libphx = require('libphx').lib
+
+function onDef_TaskQueue_t(t, mt)
+    mt.__index.sendTask = function(self, workerId, data)
+        local taskIdPtr = libphx.TaskQueue_SendTask(self, workerId, ffi.gc(PayloadConverter:valueToPayload(data, true), nil))
```

Comment:
I´d add a comment here that refers to the discussion on discord

Reply:
done